src/daily_score.rs

```diff
@@ -65,8 +65,8 @@ impl DailyScore {
     }
 
     pub fn parse(daily_score_string: &str) -> Result<Self, ParseError> {
-        let spaced_separator = format!(" {} ", crate::JOURNAL_SEPARATOR);
-        let mut slice = daily_score_string.splitn(4, spaced_separator.as_str());
+        let spaced_separator = format!(" {}", crate::JOURNAL_SEPARATOR);
+        let mut slice = daily_score_string.splitn(4, &spaced_separator).map(str::trim);
```

Comment:
I guess that does not worth it - use a separator, drop tests that are expecting to fail without a whitespace around separator